frontend/src/index.js

```diff
@@ -44,7 +45,7 @@ function canStart( config, hostname ) {
 	}
 
 	// Check if user opened page without required permissions.
-	if ( !document.getElementById( 'partial-discussion-header' ) ) {
+	if ( document.querySelectorAll( 'img.js-plaxify' )[ 1 ] && document.querySelectorAll( 'img.js-plaxify' )[ 1 ].alt.startsWith( '404' ) ) {
```

Comment:
What is the `img.js-plaxify` item? You could describe it somehow. And can't we use the first matched element?

Reply:
I think also that having only `document.querySelectorAll( 'img.js-plaxify' )[ 1 ].alt.startsWith( '404' )` might quickly break after some minor GH site changes.

Reply:
`img.js-plaxify` is a part of the 404 page. This page contains an octocat graphic built with some `img` tags. The second element of this array is the only which contains information about 404 error. I'll add a comment with explanation in the code.

Reply:
But are `img.js-plaxify` elements only present to the `Page not found`? If yes then we can just check the `!!document.querySelector( 'img.js-plaxify' )`

Reply:
Actually, any error leading to this page should remove the user from the collaboration. 404 is only one of them.

Reply:
> But are `img.js-plaxify` elements only present to the `Page not found`? If yes then we can just check the `!!document.querySelector( 'img.js-plaxify' )`

That should be enough.

Reply:
But there's no need to double negation. `document.querySelector( 'img.js-plaxify' )` returns `null` if element doesn't exist.

Reply:
yep, the `!!` only converts it to boolean.